### PR TITLE
Set all custom failsafe channels at once

### PR DESCRIPTION
**Pull request by raphaelcoeffic**
Implements #5820.

### CODE REVIEW

**Comment by raphaelcoeffic** [comment]
For now, it is only implementing the "Enter long-press" version as proposed by @kilrah.

While on the custom failsafe screen, each individual channel can already be set to current value by using enter long press while editing the value. The question is: should we have a special menu item on that screen, similar to "subtrim -> trim" menu in outputs? I think some space could be made for that menu item on Horus, and possibly X9D/E, but the X7 screen is really too small for that, and would require some special scrolling instead of paging (sorry, I'm too lazy for that one...).

Any other suggestion?

**Comment by raphaelcoeffic** [comment]
Just added such a menu item for horus:
screen shot 2018-04-21 at 13 02 48

**Comment by 3djc** [comment]
output uses >>, not ->, but on x9, x7 it is => so we probably need to bring consistentcy Which also means the BW screens need it too :) (sorry it falls on you !)

**Comment by raphaelcoeffic** [comment]
TODOs:
- [x] add "Outputs => failsafe" to X9.
- [x] add "Outputs => failsafe" to X7.

Screenshots:
screen shot 2018-04-21 at 13 09 51
screen shot 2018-04-21 at 12 37 47

File: radio/src/gui/128x64/model_setup.cpp [hunk 1]
```diff
@@ -1200,9 +1200,18 @@ void menuModelSetup(event_t event)
             }
           } else if (menuHorizontalPosition == 1) {
             s_editMode = 0;
-            if (moduleData.failsafeMode == FAILSAFE_CUSTOM && event == EVT_KEY_FIRST(KEY_ENTER)) {
-              g_moduleIdx = moduleIdx;
-              pushMenu(menuModelFailsafe);
+            if (moduleData.failsafeMode == FAILSAFE_CUSTOM) {
+                if (event == EVT_KEY_LONG(KEY_ENTER)) {
```
Comment (bsongis) [inline radio/src/gui/128x64/model_setup.cpp]: indentation issue

**Review by bsongis — APPROVED** [review]

File: radio/src/gui/128x64/model_setup.cpp (hunk 1 above)
Comment (raphaelcoeffic) [reply radio/src/gui/128x64/model_setup.cpp]: done!

**Comment by bsongis** [comment]
Thanks!

**Pull request merged**